infrastructure/service-bus.tf

```diff
@@ -8,7 +8,6 @@ resource "azurerm_servicebus_namespace" "main" {
 
   sku                          = var.service_bus_config.sku
   capacity                     = var.service_bus_config.capacity
-  zone_redundant               = var.service_bus_config.sku == "Premium"
```

Comment:
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/eventhub_namespace

Reply:
possibly need to look somewhere else in terraform to replace this